ansible/roles/windows/ipa-ad-data/tasks/ad-root.yml

```diff
@@ -13,6 +13,13 @@
     attributes:
       gidNumber: 10048
 
+- name: testgroup1
+  win_domain_group:
+    name: testgroup1
+    scope: global
+    attributes:
+      gidNumber: 55555555
```

Comment:
For consistency I would use a gidNumber that is roughly the same range as the existing users/groups (100xx are used for ad-root, 101xx for ad-child, 102xx for ad-tree).
This will also reduce the risk of having ids outside of the ranges.

But the real question is: do we need a gidNumber for this group? It is used in the non-posix scenario with nonposixuser1, and IMO the gidNumber is not needed.

Reply:
@flo-renaud, thank you for the review, i did remove the gidNumber from the above group as you mentioned.
However i am unable to authenticate to freeipa.org with fedora account for now. Will check this and update the page accordingly.